WebCryptoAPI/import_export/okp_importKey.js

```diff
@@ -67,27 +67,34 @@ function testFormat(format, algorithm, keyData, keySize, usages, extractable) {
     });
 }
 
-// Test importKey/exportKey "alg" behaviours, alg is ignored upon import and alg is missing for Ed25519 and Ed448 JWK export
-// https://github.com/WICG/webcrypto-secure-curves/pull/24
+// Test importKey/exportKey "alg" behaviours (https://github.com/w3c/webcrypto/pull/401)
+// - alg is ignored for ECDH import
+// - alg is checked to be the algorith.name or EdDSA for Ed25519 and Ed448 import
```

Comment:
```suggestion
// - alg is checked to be the algorithm.name or EdDSA for Ed25519 and Ed448 import
```
Also, this check doesn't actually seem to be tested yet, right? I.e. we should try to import a key with a different value and assert that it fails.

Reply:
I don't believe it is for ECDSA or RSA either. Would make sense to do a pass over the tested failure fixtures. I'm not up for that at the moment tho

Reply:
OK, perfectly fair, but then maybe we can add a TODO to make it clear this item isn't actually tested yet?

Reply:
I'd rather have something, either more explicit fixtures for the test cases or either the text shown for each test case, to see clearly that we are testing that the "alg" member if ignored and missed for the Ed25519 and Ed448. 

In the previous version of the test we had the "Good parameters with ignored alg" text, which I admit could be better. However, the current version is more obscure, IMHO.